### ISSUE TITLE
Indentation keeps increasing

### ISSUE THREAD

**Issue by stayallive** [Stillat/blade-parser-typescript#96]
In my current blade files I have the following:

```php
@section('content')
    <x-tw::page-header :breadcrumbs="[
                [
                    'title' => 'Shortener',
                ],
        ]">
@endsection
```

The problem is that every run of prettier the indentation if the inner array keeps increasing 😅

After about 5 runs I end up with:

```php
    <x-tw::page-header
        :breadcrumbs="[
                                        [
                                            'title' => 'Shortener',
                                        ],
        ]"
    >
```

That is less than ideal obviously! As far as I know my syntax is working just fine but maybe multi-line attributes like that are a little unconventional.

**Comment by JohnathonKoster** [comment]
Thanks for the report! I've released `2.1.10` to address this issue

**Comment by stayallive** [comment]
🔥 that was _very_ quick! Thanks, I've just confirmed and it's working beautiful!
